glean-core/src/storage/mod.rs

```diff
@@ -231,4 +231,31 @@ mod test {
             StorageManager.snapshot_experiments_as_json(glean.storage(), "glean_internal_info");
         assert!(empty_snapshot.is_none());
     }
+
+    // Experiment's API tests: the next test comes from glean-ac's
```

Comment:
I guess with glean-ac gone for good, it doesn't make sense to keep these comments around.